packages/@webex/plugin-cc/src/services/task/TaskManager.ts

```diff
@@ -0,0 +1,101 @@
+import EventEmitter from 'events';
+import {ICall, LINE_EVENTS} from '@webex/calling';
```

Comment:
We discussed about re-exporting these from WebCallingService right? Could we add a TODO here?

Reply:
This is supposed to be taken care of in the separate story I created for conditionally adding @webex/calling package and there we can take care of re-exporting. Adding a TODO is also tricky because I will have to add it in multiple places. If that's ok, I will add